condensation-forum/application.py

```diff
@@ -301,6 +314,117 @@ def logoutCallback():
     print("User signed out: " + user["name"], file=sys.stderr)
 
 
+@application.route('/file-delete', methods=['POST'])
+@authManager.requireAuthentication
+def fileManagerDeleteHander():
+    user = authManager.getUserData()
+    fid = int(request.form['file'])
+    
+    if not user:
+        abort(403)
+    id = user['id']
+
+    #delete the file by Cloud_key in AWS S3
+    with dataSessionMgr.session_scope() as dbSession:
+        file1 = query.getFileById(dbSession,fid)
+        file1 = query.extractOutput(file1)
+    
+    key = file1['cloud_key']
+    try:
+        s3client.delete_object(Bucket=bucket_name,Key=key)
+    except Exception as e:
+        print("Something happen: ",e)
```

Comment:
you should also `flash` a message to the user if this fails

Reply:
Fixed